### PR TITLE
Update Pax Documentation with instructions of additional configurations

### PR DESCRIPTION
**Pull request by ashors1**
- Updates example run commands to reflect improved configurability of scripts 
- Adds some information about relevant hyperparameters that users may want to override

### CODE REVIEW

File: rosetta/rosetta/projects/pax/README.md [hunk 1]
```diff
@@ -52,44 +51,54 @@ Note that when training with The Pile dataset, you must provide the `TFDS_DATA_D
 
 ### Quick Runs
 #### Interactive: Single Node
-See [run_pile_singlenode.sh](https://github.com/google/paxml/blob/main/paxml/contrib/gpu/scripts_gpu/run_pile_singlenode.sh) for an example of training a 126m model on a single node using The Pile. Once inside of your container, this script can be run interactively using the following command: 
+See [run_pile_singlenode.sh](https://github.com/google/paxml/blob/main/paxml/contrib/gpu/scripts_gpu/run_pile_singlenode.sh) for an example of training a 126m model on a single node GPUs using The Pile. Once inside of your container, this script can be run interactively using the following command:
```
Comment (nouiz) [inline rosetta/rosetta/projects/pax/README.md:54]: single node GPUs -> single node with GPUs?

File: rosetta/rosetta/projects/pax/README.md [hunk 2]
```diff
@@ -52,44 +51,54 @@ Note that when training with The Pile dataset, you must provide the `TFDS_DATA_D
...
+
+We provide a list of some of the frequently overridden hyperparameters, and an explanation of each, below:
+- `ICI_MESH_SHAPE`: This refers to the parallelism strategy used on chips connected by a fast network (e.g. NVLink). `ICI_MESH_SHAPE` typically has 3 dimensions, `[replica, data, model]`, corresponding to data parallelism, fully-sharded data parallelism (ZeRO-3), and tensor parallelism, respectively. To use pure data parallelism, you should set `ICI_MESH_SHAPE` to `[NUM_GPUS, 1, 1]`.
+- `DCN_MESH_SHAPE`: This refers to the parallelism strategy for machines connected by a datacenter network. This is the generally parallel strategy used _across_ nodes.  
+- `PERCORE_BATCH_SIZE`: This is the batch size loaded by each worker _prior_ to sharding the data according to the parallel strategy. We should always have that `GLOBAL_BATCH_SIZE = PERCORE_BATCH_SIZE * NUM_GPUS`, regardless of the parallel strategy. Note that a consequence of this is that `PERCORE_BATCH_SIZE` will not always equal `MICROBATCH_SIZE`, particularly when using tensor parallelism (TP). If using 2-way TP, for example, `MICROBATCH_SIZE` will be twice the `PERCORE_BATCH_SIZE`.
```
Comment (nouiz) [inline rosetta/rosetta/projects/pax/README.md:92]: Should we explain that PERCORE_BATCH_SIZE can be a fraction and what it does in that case?
Do we use that ourself?

Comment (ashors1) [reply rosetta/rosetta/projects/pax/README.md]: We don't use fractional percore batch sizes in any of our configs, but I added a quick comment on it for clarity

File: rosetta/rosetta/projects/pax/README.md [hunk 3]
```diff
@@ -52,44 +51,54 @@ Note that when training with The Pile dataset, you must provide the `TFDS_DATA_D
...
+
+Paxml uses [Fiddle](https://github.com/google/fiddle/tree/main) for configuring hyperparameters. To overwrite an existing hyperparameter from the command line, use the following syntax: 
+```
+--fdl.<PARAM_NAME>=<NEW_VALUE>
+```
+For example, in our `*.sh` scripts, we override the default values of `FPROP_DTYPE`, `ICI_MESH_SHAPE`, and `PERCORE_BATCH_SIZE`. 
+
+We provide a list of some of the frequently overridden hyperparameters, and an explanation of each, below:
+- `ICI_MESH_SHAPE`: This refers to the parallelism strategy used on chips connected by a fast network (e.g. NVLink). `ICI_MESH_SHAPE` typically has 3 dimensions, `[replica, data, model]`, corresponding to data parallelism, fully-sharded data parallelism (ZeRO-3), and tensor parallelism, respectively. To use pure data parallelism, you should set `ICI_MESH_SHAPE` to `[NUM_GPUS, 1, 1]`.
```
Comment (terrykong) [inline rosetta/rosetta/projects/pax/README.md:90]: minor: just to be consistent with pax's docs, should we say `[data, fsdp, model]`? It also looks odd that data=1 in pure data parallelism

**Review by terrykong — APPROVED** [review]

**Pull request merged**